src/main/groovy/org/pih/warehouse/monitoring/GrailsHttpTransactionNameProvider.groovy

```diff
@@ -0,0 +1,23 @@
+package org.pih.warehouse.monitoring
+
+import io.sentry.protocol.TransactionNameSource
+import io.sentry.spring.tracing.TransactionNameProvider
+import javax.servlet.http.HttpServletRequest
+
+/**
+ * Defines how Sentry transaction names for HTTP requests should be structured.
+ */
+class GrailsHttpTransactionNameProvider implements TransactionNameProvider {
+
+    String provideTransactionName(HttpServletRequest var1) {
+        // If we wanted our logs to look more like HTTP Requests, we could have done the following instead, but
+        // we decided that "invoiceApi/list" is easier to debug than "GET /openboxes/api/invoices".
+        //return "${var1.method} ${var1.requestURI}"
+        return "${var1.getAttribute("org.grails.CONTROLLER_NAME_ATTRIBUTE")}/${var1.getAttribute("org.grails.ACTION_NAME_ATTRIBUTE")}"
+    }
+
+    TransactionNameSource provideTransactionSource() {
+        // Do what SpringMvcTransactionNameProvider does, which is the provider that Sentry normally uses.
+        return TransactionNameSource.ROUTE
```

Comment:
I don't know if this really matters, but is this accurate? I feel like custom would be more accurate since it doesn't seem like we're dealing with the parameters at all. 

https://javadoc.io/static/io.sentry/sentry/6.31.0/io/sentry/protocol/TransactionNameSource.html

Reply:
Yeah good point. I originally had it using the URI approach, so this made more sense, but since we switched I think maybe COMPONENT makes the most sense since it's based off a Grails controller and method?